desk/lib/channel-utils.hoon

```diff
@@ -649,7 +649,41 @@
       &+[*reply-seal:c ~[%inline 'Unknown comment']~ ~nul *@da]
     ?:  ?=(%| -.u.reply)  u.reply
     &+(suv-reply-2 p.plan +.u.reply)
-  [%channel-said-2 !>(`said:v9:c`[nest %reply p.plan reply])]
+  [%channel-said-2 !>(`said:v9:c`[nest [%reply p.plan reply]])]
+::
+++  said-4
+  |=  [=nest:c =plan:c posts=v-posts:c group=flag:gv]
+  ^-  cage
+  =/  post=(unit (may:c v-post:c))  (get:on-v-posts:c posts p.plan)
+  ?~  q.plan
+    =/  post=(may:c simple-post:c)
+      ?~  post
+        :-  %&  ::TODO  should eventually just unitize $reference
```

Comment:
If you're open to including that change here this might be a good point to do it at, since we're doing a whole new mark/type anyway.

Would probably `+$  said-response  [group=flag:gv =nest ref=(unit reference)]` or similar. Would only need to slim down the code here to not do the dummy content, and the corresponding serialization/client-side processing changes.